lib/wiki_course_edits.rb

```diff
@@ -22,7 +23,7 @@ def initialize(action:, course:, current_user:, **opts)
   # set of participants, articles, timeline, and other details.
   # It simply overwrites the previous version.
   def update_course(delete: false)
-    return unless @course.submitted && @course.slug
```

Comment:
We still need a check for unsubmitted courses for Wiki Ed Dashboard.

This is something we could delegate to the specific Course subclasses via something like `course.wiki_course_page_enabled?`. That would check for submitted  status for ClassroomProgramCourse, and return true for others. I'm not sure why the check for a slug is here, as it shouldn't be possible to have a course without one.